flixel/addons/api/FlxGameJolt.hx

```diff
@@ -423,17 +428,21 @@ class FlxGameJolt
 	/**
 	 * Retrieve the high scores from this game's remote data. If not authenticated, leaving Limit null will still return the top ten scores. Requires initialization.
 	 *
-	 * @see		http://gamejolt.com/api/doc/game/scores/fetch/
+	 * @see		https://gamejolt.com/api/doc/game/scores/fetch/
+	 * @param	TableID		The table you want to pull from
 	 * @param	Limit		The maximum number of scores to retrieve. Leave null to retrieve only this user's scores.
 	 * @param	CallBack	An optional callback function. Will return a Map<String:String> whose keys and values are equivalent to the key-value pairs returned by GameJolt.
 	 */
-	public static function fetchScore(?Limit:Int, ?Callback:Dynamic):Void
+	public static function fetchScore(?TableID:Int, ?Limit:Int, ?Callback:Dynamic):Void
```

Comment:
is TableID optional? what happens if you don't specify it?

If it is optional it should be added after `Limit` for better backwards compatibility

Reply:
Also, if optional the Doc should explain what happens if ommitted, as well